scripts/cleanup-docker.sh

```
@@ -0,0 +1,157 @@
+    PRUNE_CMD="$PRUNE_CMD --all"
+fi
+
+# Add force flag for non-interactive mode
+if $DRY_RUN; then
+    log_info "Would execute: $PRUNE_CMD -f"
+    echo ""
+    log_warn "Dry run complete - no changes made"
+else
+    log_info "Executing: $PRUNE_CMD"
+    echo ""
+    eval $PRUNE_CMD
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)